src/JuliaInterpreter.jl

```diff
@@ -11,7 +11,7 @@ using UUIDs
 using Random.DSFMT
 using InteractiveUtils
 
-export @enter, @make_stack, @interpret, Compiled, JuliaStackFrame
+export @enter, @interpret, Compiled, JuliaStackFrame
```

Comment:
Hasn't `@enter` already moved to Debugger?

Reply:
Good catch, removing it here as well.